test/testmathlib.cpp

```diff
@@ -312,6 +313,11 @@ class TestMathLib : public TestFixture {
         ASSERT_EQUALS_DOUBLE(0.0   , MathLib::toDoubleNumber("-0.0"));
         ASSERT_EQUALS_DOUBLE(0.0   , MathLib::toDoubleNumber("+0.0"));
 
+        // Ticket #6795
+        std::setlocale(LC_NUMERIC, "de_DE");
```

Comment:
It is suggested here (http://www.thecodingforums.com/threads/how-to-make-std-strtod-not-depend-on-locale.546559/) to use the "C" locale:

    std::setlocale(LC_NUMERIC, "C");

however this does not feel like a good solution. what if this mess with some library we use (currently: tinyxml,pcre,qt). what if somebody would use cppcheck library and this setlocale is overridden.

is there a better solution?